samples/snippets/csharp/VS_Snippets_Remoting/System.Net.Authentication/CS/custombasicauthentication.cs

```diff
@@ -124,7 +124,7 @@ private static void getPage(String url)
 
     // The displayPageContent method display the content of the
     // selected page.
-    private static void displayPageContent(Stream ReceiveStream)
+    private static void DisplayPageContent(Stream ReceiveStream)
     {
       // Create an ASCII encoding object.
       Encoding ASCII = Encoding.ASCII;

```

Comment:
This probably should be ascii instead of ASCII.
The indentation also uses 2 spaces instead of 4 spaces.

I think it's better to address all stylistic issues.